src/components/quantity-lessons-card/QuantityLessonsChart.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react'
+import { Bar } from 'react-chartjs-2'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+} from 'chart.js'
+import { Box } from '@mui/material'
+import useBreakpoints from '~/hooks/use-breakpoints'
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend)
+
+const data = {
+  labels: [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC'
+  ],
+  datasets: [
+    {
+      label: 'Lessons',
+      data: [5, 5, 10, 10, 12, 15, 14, 14, 10, 10, 12, 12],
+      backgroundColor: '#79B260',
+      borderColor: '#79B260',
+      borderWidth: 1,
+      barThickness: 28
+    }
+  ]
+}
+
+const options = {
```

Comment:
This object directly belongs to component properties, but not a hardcoded value which is used as input data for the component. That's why it is better to leave it at the same place as the component

Reply:
I still don't understand why we can't move these constant values to constants.tsx file, because we store constants in separate files to avoid cluttering files with components

Reply:
you can move these constants to another file and just import them, because now the component is heavily cluttered